### PR TITLE
interactive bots: Run the helloworld bot for sColin16

### PR DESCRIPTION
**Pull request by sColin16**

### CODE REVIEW

File: interactive-bots/helloworld/sColin16/notes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+bot could not run. Thus, I had to open a second console to run the bot while the zulip test instance ran on another console
+
+3. **Working with feature branches** - In order to push the images and this note page to the github repository in the appropriate feature 
+branch, I had to review git commands to work with the branches.
+
+4. **Learning Markdown** - I wanted to make the notes file look interesting, so I quickly learned some basic markdown syntax :smile:
+
+Screenshot notes
+================
+* **Zulipbot-responses.png** - This screenshot shows the helloworld responding to the messages sent to it
```
Comment (roberthoenig) [inline interactive-bots/helloworld/sColin16/notes.md]: Always finish phrases with a dot when you start them capitalized.

File: interactive-bots/helloworld/sColin16/notes.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Problems I faced:
+=================
+
+1. **Obtaining the zuliprc configuration on the dev enviornment** - Although I was able to download the file onto my local file system, I was 
+unable to use the configuration file in the dev enviornment, because it was not there. To solve this problem, I used the scp command from my
```
Comment (roberthoenig) [inline interactive-bots/helloworld/sColin16/notes.md]: We try to keep documentation lines <= 80-100 characters; can you shorten some of the very long lines?

File: interactive-bots/helloworld/sColin16/notes.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Problems I faced:
+=================
+
+1. **Obtaining the zuliprc configuration on the dev enviornment** - Although I was able to download the file onto my local file system, I was 
+unable to use the configuration file in the dev enviornment, because it was not there. To solve this problem, I used the scp command from my 
+local enviornment, to send the configuration file to the development enviornment.
+
+2. **Running the bot from a seperate console** - Initially, I attempted to run the bot without the zulip test instance running. Of course, the
```
Comment (roberthoenig) [inline interactive-bots/helloworld/sColin16/notes.md]: We usually call the Zulip test instance the "Zulip development environment", or simply "Zulip dev env". Can you use that wording?

File: interactive-bots/helloworld/sColin16/notes.md (hunk 2 above)
Comment (roberthoenig) [inline interactive-bots/helloworld/sColin16/notes.md]: typo: s/enviornment/environment

File: interactive-bots/helloworld/sColin16/notes.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+Problems I faced:
+=================
+
+1. **Obtaining the zuliprc configuration on the dev enviornment** - Although I was able to download the file onto my local file system, I was 
+unable to use the configuration file in the dev enviornment, because it was not there. To solve this problem, I used the scp command from my 
+local enviornment, to send the configuration file to the development enviornment.
```
Comment (roberthoenig) [inline interactive-bots/helloworld/sColin16/notes.md]: Same typo :)

File: interactive-bots/helloworld/sColin16/notes.md (hunk 3 above)
Comment (roberthoenig) [inline interactive-bots/helloworld/sColin16/notes.md]: typo: s/seperate/separate

**Comment by sColin16** [comment]
Will do. Is it O.K. If I push a second commit to make those changes?

**Comment by jajodiaraghav** [comment]
Awesome @sColin16 
This looks good to me. Merging and Approving 👍

**Pull request merged**